src/mon/MonCommands.h

```diff
@@ -235,6 +235,10 @@ COMMAND("report name=tags,type=CephString,n=N,req=false", \
 	"mon", "r", "cli,rest")
 COMMAND("quorum_status", "report status of monitor quorum", \
 	"mon", "r", "cli,rest")
+COMMAND("mon_metadata name=rank,type=CephInt,range=0",
```

Comment:
inconsistent use of underscores vs spaces in commands (including pre-existing commands) -- I'd suggest all the new commands should use spaces.